src/main/app/src/App.js

```diff
@@ -251,8 +253,13 @@ const App = () => {
   const isFetching = useIsFetching();
   const [sideMenuKey, setSideMenuKey] = useState(1);
 
+  const focusRef = useRef(null);
   const chatIsVisible = useChat();
 
+  useEffect(() => {
```

Comment:
https://blog.octo.com/why-you-should-use-uselayouteffect-when-tinkering-with-focus/
```suggestion
  useLayoutEffect(() => {
```